qiskit/quantum_info/states/stabilizerstate.py

```diff
@@ -231,10 +233,11 @@ def expectation_value(self, oper, qargs=None):
         # Check if there is a stabilizer that anti-commutes with an odd number of qubits
         # If so the expectation value is 0
         for p in range(num_qubits):
-            stab = self.clifford.stabilizer
+            stab_x = self.clifford.stabilizer_x_tableau()
+            stab_z = self.clifford.stabilizer_z_tableau()
```

Comment:
This is why I'm in favour of method calls, not properties.  It's clear here that we're doing a whole load of useless work - we should move `stab_x` and `stab_z` outside the loop.